### PR TITLE
Remove github repository configurations as we have migrated these to new tdr-terraform-github repo

### PR DESCRIPTION
**Pull request by vimleshtna**

### CODE REVIEW

File: root_github.tf [hunk 1]
```diff
@@ -67,84 +67,6 @@ module "github_oidc_provider" {
   common_tags = local.common_tags
```
Comment (TomJKing) [inline root_github.tf:67]: The `locals` resource block on line 6 can be removed as its no longer used.

**Review by TomJKing — COMMENTED** [review]
Looks good. Thanks

The `common_ssm_parameters` module should be removed from the `root.tf` as well, as this is now being set in the `tdr-terraform-github`

File: root_github.tf (hunk 1 above)
Comment (vimleshtna) [reply root_github.tf:55]: Yeah, I have removed it and removed the `common_ssm_parameters` module as well, thanks

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

**Pull request merged**